src/panels/experimental-ui/hui-camera-preview-card.js

```diff
@@ -0,0 +1,74 @@
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import computeDomain from '../../common/entity/compute_domain.js';
+
+import '../../cards/ha-camera-card.js';
+import './hui-error-card.js';
+
+class HuiCameraPreviewCard extends PolymerElement {
+  static get properties() {
+    return {
+      hass: {
+        type: Object,
+        observer: '_hassChanged',
+      },
+      config: {
+        type: Object,
+        observer: '_configChanged',
+      }
+    };
+  }
+
+  getCardSize() {
+    return 4;
+  }
+
+  _configChanged(config) {
+    this._entityId = null;
+    if (this.childNodes.length) {
+      this.removeChild(this.childNodes[0]);
+    }
+
+    let error = null;
+    let cardConfig;
+    let tag;
+
+    const entityId = config && config.entity;
+    if (entityId) {
+      if (computeDomain(entityId) === 'camera') {
+        this._entityId = entityId;
+        tag = 'ha-camera-card';
+        cardConfig = config;
+      } else {
+        error = 'Entity domain must be "camera"';
+      }
+    } else {
+      error = 'Entity not defined in card config';
```

Comment:
For now this is fine. We might add an option to hide the card if entity not found in the future